internal/learn/future/assembly.md

```diff
@@ -0,0 +1,35 @@
+---
+id: assembly
+title: Assembly
+description: Introduction to the emerging permissionless Smart Contracts network
+keywords:
+  - smart contracts
+  - permissionless
+  - ASMB
+---
+
+![Assembly](/img/learn/future/assembly.png 'Click to see the full-size image.')
+
+In December 2021, IOTA Foundation [introduced](https://blog.assembly.sc/announcing-assembly-and-the-asmb-token/) Assembly, a permissionless Smart Contracts network. Assembly will function as an intermediate layer for permissionless and interoperable Smart Contracts. It will also have its own token called [ASMB](https://assembly.sc/token). The Token will help secure the network.
```

Comment:
```suggestion
In December 2021, IOTA Foundation [introduced](https://blog.assembly.sc/announcing-assembly-and-the-asmb-token/) Assembly, a permissionless Smart Contracts network. Assembly will function as an intermediate layer for permissionless and interoperable Smart Contracts, and will have its own token called [ASMB](https://assembly.sc/token).
```

There is too much information for a single paragraph, that's my lazy attempt to fix it.